proto/api/v1/metadata_service.proto

```diff
@@ -24,6 +24,8 @@ message Metadata {
 
     // Do not use this field. It will always contain "VALID"
     LicenseStatus license_status = 4 [deprecated = true];
+
+    bool is_managed_service = 5;
```

Comment:
Subjective opinion, but I would suggest to stick with `is_managed_central` name instead of `*_service` here.
The reason is that "managed service" is broad and can mean multiple things, moreover it can give a false impression that the entire service is managed including the secured clusters.
"Managed central", in contrast, says only about central and is not misleading. Moreover, it's consistent with `ROX_MANAGED_CENTRAL` environment variable name. Also, knowing that central is managed seems to be sufficient for currently intended UI use.

Secondly I would suggest adding comments that explain this attribute. Something along the lines of:

```suggestion
	// is_managed_central is set to true in case central is running in the Red Hat cloud and is managed by Red Hat. Otherwise it will be false.
    bool is_managed_central = 5;
```